app/Http/routes.php

```diff
@@ -113,6 +113,17 @@
     Route::get('api/countries/{country_id}', 'CountryController@getCountry')->where('country_id', '[0-9]+');
     Route::put('api/countries/{country_id}', 'CountryController@updateCountry')->where('country_id', '[0-9]+');
     Route::post('api/countries', 'CountryController@createCountry');
+
+    Route::get('api/circles/', 'CircleController@getCircles');
+    Route::get('api/circles/{circle_id}', 'CircleController@getCircle')->where('circle_id', '[0-9]+');
+    Route::get('api/circles/{circle_id}/children', 'CircleController@getCircleChildren')->where('circle_id', '[0-9]+');
+    Route::get('api/circles/{circle_id}/parents', 'CircleController@getCircleParents')->where('circle_id', '[0-9]+');
```

Comment:
Well, you can consider the parent of your parent to also be your parent > allows multiple parents.
However, especially as the default is having 1 parent, I agree with you.